docs/source/tutorials/qubo_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>typo: 'let's define a function brutally calculate the costs (classical methods)' -&gt; 'let's define a function to classically brute-force calculate all possible combinations of stocks and their associated cost functions.' </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>